src/cljs/replumb/repl.cljs

```diff
@@ -591,6 +593,24 @@
       (call-back (common/wrap-success (s/join (map #(with-out-str (repl/print-doc %)) ms))))
       (call-back (common/wrap-success "nil")))))
 
+(declare read-eval-call)
```

Comment:
Mmm if you need the whole function we can add a two parameter function that calls read-eval-call but I would not add :filename to it...thoughts?